sasjs-tests/src/testSuites/Basic.ts

```diff
@@ -47,7 +47,9 @@ export const basicTests = (
         'Should fail on first attempt and should log the user in on second attempt',
       test: async () => {
         await adapter.logOut()
+        await sleep(1000)
```

Comment:
What purpose is this serving? Will it work if the server takes longer to respond?

Reply:
Seems that promise is resolved sooner then the login/out requests. So calling it without pause makes it unstable, sometimes fail and hang on one of the requests. If server is slower effect should be the same, purpose is to slow down calling of next function `after` the promise is resolved. No matter the speed of server, promise should be resolved in relative to it.